src/applications/personalization/view-dependents/components/ViewDependentsList/ViewDependentsListItem.jsx

```diff
@@ -2,62 +2,37 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const ViewDependentsListItem = props => (
-  <div className="vads-l-col--12 vads-u-background-color--gray-lightest vads-u-margin-top--0 vads-u-margin-bottom--2 vads-u-padding-top--1 vads-u-padding-bottom--2 vads-u-padding-x--2">
+  <div className="vads-l-col--12 vads-u-background-color--gray-lightest vads-u-margin-top--0 vads-u-margin-bottom--2 vads-u-padding-top--2 vads-u-padding-bottom--3 vads-u-padding-x--2">
     <div className="vads-l-row">
-      <div className="vads-l-col--9">
-        <p className="vads-u-font-weight--bold vads-u-margin-top--0p25 vads-u-margin-bottom--0 vads-u-margin-x--0 vads-u-font-size--lg">
-          {props.name}
+      <p className="vads-u-font-weight--bold vads-u-margin-top--0p25 vads-u-margin-bottom--0 vads-u-margin-x--0 vads-u-font-size--lg">
+        {props.name}
+      </p>
+    </div>
+    <div className="vads-l-row">
+      <p className="vads-u-margin--0">
+        <strong>Relationship:</strong> {props.spouse ? 'Spouse' : 'Child'}
+      </p>
+    </div>
+    <div className="vads-l-row">
+      {props.social ? (
+        <p className="vads-u-margin-top--0 vads-u-margin-bottom--0">
+          <strong>SSN:</strong> {props.social}
         </p>
```

Comment:
I could see these p tags potentially running into the same issue we had with the content cards for rated disabilities and the use of description lists. Should we consider changing the markup now, or just run with what we have here until we hear otherwise?

Reply:
Yep, its probably good to just head this off now. Fixed.